README

```diff
@@ -10,3 +10,10 @@ mkdir build
 cd build
 cmake ../src
 make -j
+
+
+License
+-------
+list.h is distributed under the GPL-2.0 license.  Other source files are
+distributed under the MPL 1.1 license.  See COPYING.GPL and COPYING.MPL
```

Comment:
Maybe it is worth pointing out that the other source files are licensed under a mix of MPL 1.1, GPL 2.0 and LGPL 2.1. Because technically MPL 1.1 is not compatible with the GPL (MPL 2 is). To make it clear that users of the module are able to select GPL 2.0.

Reply:
Good point.  I have pushed a fixup commit to clarify it: f1c182032c155e2d7f53995a941a00df8aba52ca